### ISSUE TITLE
Providing options to the compressors using the CLI

### ISSUE THREAD

**Issue by rmueller78** [srod/node-minify#395]
Is there a way to provide options or arguments to a compressor when using the CLI? The usage information (--help) mentions only options for input file (-i), output file (-o) and compressor(s) (-c). Is there any way to provide additional arguments or options to a compressor either via an option (-something) or a configuration file?

**Comment by srod** [comment]
Not at this moment, I am working on it and I will release it as soon as possible.

**Comment by rmueller78** [comment]
Thank you for your quick reply. Also, excellent job with node-minify so far!

**Comment by srod** [comment]
Thank you too ! 👍

**Comment by srod** [comment]
I've released [3.3.0](https://github.com/srod/node-minify/releases/tag/3.3.0) with the `option` parameter.

You have to pass them as a JSON: `{"warnings": true, "mangle": false}`

Example:

```bash
node-minify --compressor uglifyjs --input 'input.js' --output 'output.js' --option '{"warnings": true, "mangle": false}'
```
